tests/sls/homu/config_permissions.py

```diff
@@ -0,0 +1,26 @@
+import os
+import pwd
+import stat
+
+from tests.util import Failure, Success
+
+
+def get_owner(filename):
+    return pwd.getpwuid(os.stat(filename).st_uid).pw_name
+
+
+def is_world_readable(filename):
+    st = os.stat(filename)
+    return bool(st.st_mode & stat.S_IROTH)
+
+
+def run():
+    for root, directories, filenames in os.walk('/home/servo/homu/'):
```

Comment:
This will also check files like `.profile` for the homu user. I think it's OK if we only check permissions on Homu's `cfg.toml` file (and the database if it exists).